app/containers/DatasetPage.js

```diff
@@ -305,13 +426,29 @@ class DatasetPage extends Component {
     )
 
     const BuildDatasetButton = withCurrentScheduler(Button)
+    let bidsBlockStyle = {display:'none'}
+    let uploadBlockStyle = {display:'none'}
+    let fetchBlockStyle = {display:'none'}
+    const format = configuration.getIn(['format'])
+    format === 'bids' ? bidsBlockStyle = {} : bidsBlockStyle
+    format === 'upload' ? uploadBlockStyle = {} : uploadBlockStyle
+    format === 'fetch' ? fetchBlockStyle = {} : fetchBlockStyle
```

Comment:
Thank you for the advice! Why is it important to be a constant?

Reply:
(I forgot to mention that the new constant pattern is now applied)

Reply:
In this case, it is not much about the constant, butt because it is hard to read in comparison to this:
```
const bidsBlockStyle = format === 'bids' ? {} : {display:'none'};
```
and it is transformed into a constant since it will not change anymore. The lesser the number of things that can change, the easier to control.